src/store/data/actions/joinCommunity.ts

```diff
@@ -100,16 +100,9 @@ export default async ({ joiningLink }: Payload): Promise<void> => {
       // We add a default channel that is a reference to
       // the community itself. This way we can utilize the fractal nature of
       // neighbourhoods. Remember that this also need to happen in create community.
-      dataStore.addLocalChannel({
-        perspectiveUuid: neighbourhood.uuid,
-        channel: {
-          perspectiveUuid: neighbourhood.uuid,
-          hasNewMessages: false,
-          feedType: FeedType.Signaled,
-          notifications: {
-            mute: false,
-          },
-        },
+      await dataStore.createChannel({
```

Comment:
Will this call throw an exception since the Home channel is already in the link language?

Reply:
It wont throw an exception but could result in multiple links being created, I think we can assume that if you join a Flux neighbourhood then it should have already had a home channel created and thus this call is not needed @fayeed

Reply:
Also due to the upcoming link language changes, once you join a community and start syncing link data, the links will arrive to our UI via link signals so we should get the channel links once they become available to an agent.

Reply:
Yes, this was not intended I am pushing a fix.